tools/graphmap/graphmap_align.xml

```diff
@@ -0,0 +1,65 @@
+<tool id="graphmap_align" name="Mapper" version="0.5.2">
+    <description>for long, error-prone reads, like Nanopore ONT and PacBio.</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <command detect_errors="exit_code">
+    <![CDATA[
+        ln -s '$r' ref.${r.ext} && 
+        ln -s '$d' reads.${d.ext} && 
+
+        graphmap align  
+        -r ref.${r.ext} 
+        -d reads.${d.ext}
+        -o output.sam 
+        #if $preset:
+            -x $preset
+        #end if
+        -t \${GALAXY_SLOTS:-4} 
+    ]]></command>
+    <inputs>
+        <param argument="-r" type="data"  format="fasta,fastq" label="reference sequence" />
+        <param argument="-d" type="data"  format="fasta,fastq" label="reads" />
+        <param argument="-u" type="boolean" truevalue="-u" falsevalue="" label="SAM in the order of input reads" />
```

Comment:
The label could be more clear. Is this an input sorted SAM file? Wouldn't it be better to output the BAM natively?

Reply:
It does only report sam. With this option the sam will be sorted. I will update the desc. :+1:

Reply:
we could easily depend on samtools and do the conversion on our own, if needed to both coordinate-sorted-bam and name-sorted bam

Reply:
Just to be clear, I'm taking about adding a `&& samtools view -b` thing at the end of the command.

Reply:
Aha this sound useful and neeed anyway. ok

Reply:
So like this: `&&        samtools view -b > output.bam`
Also sorting?

Reply:
https://github.com/galaxyproject/tools-iuc/blob/57493fdc88e6128bed127993464f2a6480e2774d/tools/bwameth/bwameth.xml#L81